WordPress/Classes/Utility/PushNotificationsManager.swift

```diff
@@ -20,42 +21,77 @@ final public class PushNotificationsManager : NSObject
     
     /// Returns the shared PushNotificationsManager instance.
     ///
-    static let sharedInstance = PushNotificationsManager()
+    public static let sharedInstance = PushNotificationsManager()
```

Comment:
Actually it seems like you've added `public` in several spots.  Is there any particular reason?